cgroup/cgroup.go

```diff
@@ -82,7 +89,26 @@ func (m manager) write(path, data string) error {
 
 // generatePath generates a path within the cgroup like /<mount>/<kind>/<path (kubepods)>
 func (m manager) generatePath(kind string) string {
-	return fmt.Sprintf("%s%s/%s", m.mount, kind, m.path)
+	generatedPath := fmt.Sprintf("%s%s/%s", m.mount, kind, m.path)
+	foundPath := fmt.Sprintf("%s%s/%s", m.mount, kind, m.cgroup.Paths[kind])
```

Comment:
do we need to check that `m.cgroup.Paths[kind]` exists?

Reply:
Yep, we do! Good eye.

Reply:
Okay, so I've checked the behavior, and I don't think we need to. this is our fallback path, and if it doesn't exist we are gonna error immediately after this is called anyway. I've checked what happens if you get a string from a map[string]string if the key doesn't exist, and it will just be an empty string, so we'll log the warning appropriately and the error will occur in the expected place